DESCRIPTION

```diff
@@ -33,7 +33,8 @@ Imports:
     methods,
     mailR,
     pracma,
-    rmarkdown
+    rmarkdown,
+    markdown
```

Comment:
markdown is a package we use but wasn't included here.  It was installed previously because of a package dependency on another package. If we were missing another dependency it would have been caught when trying to install racas.